src/react/src/components/Contribute/ProductionLocationDialog.jsx

```diff
@@ -10,184 +21,318 @@ import DialogTitle from '@material-ui/core/DialogTitle';
-                            </Typography>
-                            <Typography className={classes.primaryText}>
-                                Textile or Material Production
-                            </Typography>
-                            <Typography className={classes.label}>
-                                Parent company
-                            </Typography>
-                            <Typography className={classes.primaryText}>
-                                Unifill Composite Dyeing Mills Ltd.
-                            </Typography>
                         </Grid>
-                    </Grid>
-                </DialogContent>
-                <DialogActions>
-                    <Grid container className={classes.buttonContentStyle}>
-                        <Button
-                            variant="contained"
-                            color="default"
-                            onClick={() => history.push(mainRoute)}
-                            className={classes.button}
-                        >
-                            Search OS Hub
-                        </Button>
-                        <Button
-                            variant="contained"
-                            color="secondary"
-                            onClick={() => {
-                                console.log('submit another location');
-                            }}
-                            className={classes.button}
-                        >
-                            Submit another Location
-                        </Button>
-                        <DialogTooltip
-                            text="You'll be able to claim the
-                                        location after the moderation is done"
-                            aria-label="Claim button tooltip"
-                            childComponent={claimButton(classes)}
-                        />
-                    </Grid>
-                </DialogActions>
-            </div>
-        </Dialog>
+                    </DialogContent>
+                    <DialogActions>
+                        <Grid container className={classes.buttonContentStyle}>
+                            <Button
+                                variant="contained"
+                                color="default"
+                                onClick={() => history.push(mainRoute)}
+                                className={classes.button}
+                            >
+                                Search OS Hub
+                            </Button>
+                            <Button
+                                variant="contained"
+                                color="secondary"
+                                onClick={() => {
+                                    history.push(
+                                        searchByNameAndAddressResultRoute,
+                                    );
+                                }}
```

Comment:
Please create a function that handles routing.
`onClick={handle...}`

Reply:
Done